blog/_posts/2018-09-17-Hurdles-of-a-beginning-developer.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Feeling adventurous ? become developer : choose another profession"
+layout: post
+created_at: Mon September 17 2018
+permalink: blog/2018-09-17-Hurdles-of-a-beginning-developer
+current: blog
+author: Sabine van der Eijk and Alina Leuca
+twitter: Sabin_E
+categories:
+  - student-posts
+---
+
+## Hurdles of a beginning developer
+
+So, you've done a bootcamp, congratulations you're now a programmer!! It's weird to think that now you have a developers' certificate for the whole world to see, but it still doesn't feel like you're a dev. Now you can start applying for jobs as a junior developer and dip your toes in a new project. With this new job comes your first programming experience, and the ups and downs of being a junior in a field of seniors.
+
+When we wrapped up our bootcamp, we didn't feel confident that we had a dev toolkit; we were so lucky to come across and be able to apply for a Summer of Code together. Entering a program where you can start learning by doing was a great opportunity to dip our toes into a real project and an amazing stepping-stone to an eventual job. We spent some time after our bootcamp classes checking our application and searching for coaches. We had our doubts during the process, thinking we would be so lucky if we were to be chosen, so we chose the projects that we felt comfortable with and most familiar. We reached out to the community to find coaches willing to invest time for a whole three months during the program. Alina already knew who our first coach would be, Daniel. By reaching out to reddit Amsterdam community we found our second coach Jack.
+
+<img src="/img/blog/2018/team-potatocode-first-call-01.jpg" alt="first team call">
+<div class="image-credits"> Our team on our first team potatoCode call. </div>
```

Comment:
Hi @Sveijk ! Thanks for the PR. It's always good to hear about the wonderful time our teams had during the program. :)

Minor comments here. The text you have added for each images in the blog posts are more like image description and not image credits. I think it would be good if you can add image credits in brackets or something like that. Hope that makes sense. 

Once that's fixed, we'll be raedy to publish the post. Good work. :+1: